[hueemulation] 0 published lights / sensors

Hello,
i update the openhab 2 to the version 2.5.0 S1639-1
now the status is 

```
0 published lights (see http://IP:80/api/testuser/lights)
0 published sensors (see http://IP:80/api/testuser/sensors)

```
and if i opened the page i get { } .. i only update it

```
019-07-15 21:28:27.648 [DEBUG] [ulation.internal.HueEmulationService] - REST response: null
2019-07-15 21:29:30.402 [DEBUG] [ulation.internal.HueEmulationService] - REST request GET e7914080-0d11-462f-b019-7d4eb8bab51d/lights: 
2019-07-15 21:29:30.406 [DEBUG] [ulation.internal.HueEmulationService] - REST response: null
2019-07-15 21:30:33.178 [DEBUG] [ulation.internal.HueEmulationService] - REST request GET e7914080-0d11-462f-b019-7d4eb8bab51d/lights: 
2019-07-15 21:30:33.180 [DEBUG] [ulation.internal.HueEmulationService] - REST response: null
2019-07-15 21:31:09.897 [DEBUG] [ulation.internal.HueEmulationService] - REST request GET testuser/lights: 
2019-07-15 21:31:09.900 [DEBUG] [ulation.internal.HueEmulationService] - REST response: {}
2019-07-15 21:31:35.948 [DEBUG] [ulation.internal.HueEmulationService] - REST request GET e7914080-0d11-462f-b019-7d4eb8bab51d/lights: 
2019-07-15 21:31:35.955 [DEBUG] [ulation.internal.HueEmulationService] - REST response: null

```

OK, after setting the log level to warm Alex find the devices and is able to switch on and off.
BUT she says "Light don't answer, please look to your network connection" / "Licht antwortet nicht, bitte überprüfe die Netzwerkverbindung und Stromversorgung deines Gerätes"
But she switch the light on and off

Maybe this can help 

Switch ON
```
2019-07-30 11:19:07.300 [DEBUG] [lation.internal.rest.LightsAndGroups] - sending ON to og_sz_1
2019-07-30 11:19:07.321 [WARN ] [org.eclipse.jetty.server.HttpChannel] - /api/xwcVRssJidio22nQS2KflGt0QgOprK6R47iy4QQR/lights/10/state
javax.servlet.ServletException: javax.servlet.ServletException: java.lang.IllegalStateException: Entity input stream has already been closed.
	at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:88) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.Server.handle(Server.java:505) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:370) [89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:267) [89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305) [80:org.eclipse.jetty.io:9.4.18.v20190429]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103) [80:org.eclipse.jetty.io:9.4.18.v20190429]
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:117) [80:org.eclipse.jetty.io:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:698) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:804) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: javax.servlet.ServletException: java.lang.IllegalStateException: Entity input stream has already been closed.
	at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:489) ~[?:?]
	at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:427) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:388) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:341) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:228) ~[?:?]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:873) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:542) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:71) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:146) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:548) ~[87:org.eclipse.jetty.security:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1700) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:255) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1345) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:293) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:203) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:480) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1667) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:201) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1247) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:80) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	... 15 more
Caused by: java.lang.IllegalStateException: Entity input stream has already been closed.
	at org.glassfish.jersey.message.internal.EntityInputStream.ensureNotClosed(EntityInputStream.java:228) ~[?:?]
	at org.glassfish.jersey.message.internal.InboundMessageContext.getEntityStream(InboundMessageContext.java:785) ~[?:?]
	at org.openhab.io.hueemulation.internal.HueEmulationService$LogAccessInterceptor.filter(HueEmulationService.java:100) ~[?:?]
	at org.glassfish.jersey.server.ContainerFilteringStage$ResponseFilterStage.apply(ContainerFilteringStage.java:196) ~[?:?]
	at org.glassfish.jersey.server.ContainerFilteringStage$ResponseFilterStage.apply(ContainerFilteringStage.java:163) ~[?:?]
	at org.glassfish.jersey.process.internal.Stages.process(Stages.java:171) ~[?:?]
	at org.glassfish.jersey.server.ServerRuntime$Responder.processResponse(ServerRuntime.java:442) ~[?:?]
	at org.glassfish.jersey.server.ServerRuntime$Responder.process(ServerRuntime.java:434) ~[?:?]
	at org.glassfish.jersey.server.ServerRuntime$2.run(ServerRuntime.java:329) ~[?:?]
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:271) ~[?:?]
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:267) ~[?:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315) ~[?:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297) ~[?:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:267) ~[?:?]
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:317) ~[?:?]
	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:305) ~[?:?]
	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:1154) ~[?:?]
	at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:473) ~[?:?]
	at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:427) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:388) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:341) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:228) ~[?:?]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:873) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:542) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:71) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:146) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:548) ~[87:org.eclipse.jetty.security:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1700) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:255) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1345) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:293) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:203) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:480) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1667) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:201) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1247) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:80) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	... 15 more
2019-07-30 11:19:07.750 [DEBUG] [ulation.internal.HueEmulationService] - REST request GET xwcVRssJidio22nQS2KflGt0QgOprK6R47iy4QQR/lights/10: 
2019-07-30 11:19:07.755 [DEBUG] [ulation.internal.HueEmulationService] - REST response: {"state":{"bri":254,"ct":500,"on":true,"reachable":true,"mode":"homeautomation","alert":"none"},"type":"Color temperature light","modelid":"LTW001","uniqueid":"44b2a631-db86-44e1-86a6-830d5e885584-10","manufacturername":"Philips","swversion":"66012040","friendsOfHue":true,"colorGamut":"2200K-6500K","hascolor":false,"name":"Schlafzimmer Licht","config":{"archetype":"classicbulb","function":"functional","direction":"omnidirectional"},"capabilities":{"certified":true,"streaming":{"renderer":false,"proxy":false},"control":{}}}
2019-07-30 11:19:12.671 [DEBUG] [ulation.internal.HueEmulationService] - REST request GET e7914080-0d11-462f-b019-7d4eb8bab51d/lights: 
2019-07-30 11:19:12.680 [DEBUG] [ulation.internal.HueEmulationService] - REST response: null
```
Switch OFF
```
2019-07-30 11:20:09.058 [DEBUG] [lation.internal.rest.LightsAndGroups] - sending OFF to og_sz_1
2019-07-30 11:20:09.075 [WARN ] [org.eclipse.jetty.server.HttpChannel] - /api/xwcVRssJidio22nQS2KflGt0QgOprK6R47iy4QQR/lights/10/state
javax.servlet.ServletException: javax.servlet.ServletException: java.lang.IllegalStateException: Entity input stream has already been closed.
	at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:88) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.Server.handle(Server.java:505) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:370) [89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:267) [89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305) [80:org.eclipse.jetty.io:9.4.18.v20190429]
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103) [80:org.eclipse.jetty.io:9.4.18.v20190429]
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:117) [80:org.eclipse.jetty.io:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:698) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:804) [92:org.eclipse.jetty.util:9.4.18.v20190429]
	at java.lang.Thread.run(Thread.java:748) [?:?]
Caused by: javax.servlet.ServletException: java.lang.IllegalStateException: Entity input stream has already been closed.
	at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:489) ~[?:?]
	at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:427) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:388) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:341) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:228) ~[?:?]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:873) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:542) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:71) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:146) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:548) ~[87:org.eclipse.jetty.security:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1700) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:255) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1345) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:293) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:203) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:480) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1667) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:201) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1247) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:80) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	... 15 more
Caused by: java.lang.IllegalStateException: Entity input stream has already been closed.
	at org.glassfish.jersey.message.internal.EntityInputStream.ensureNotClosed(EntityInputStream.java:228) ~[?:?]
	at org.glassfish.jersey.message.internal.InboundMessageContext.getEntityStream(InboundMessageContext.java:785) ~[?:?]
	at org.openhab.io.hueemulation.internal.HueEmulationService$LogAccessInterceptor.filter(HueEmulationService.java:100) ~[?:?]
	at org.glassfish.jersey.server.ContainerFilteringStage$ResponseFilterStage.apply(ContainerFilteringStage.java:196) ~[?:?]
	at org.glassfish.jersey.server.ContainerFilteringStage$ResponseFilterStage.apply(ContainerFilteringStage.java:163) ~[?:?]
	at org.glassfish.jersey.process.internal.Stages.process(Stages.java:171) ~[?:?]
	at org.glassfish.jersey.server.ServerRuntime$Responder.processResponse(ServerRuntime.java:442) ~[?:?]
	at org.glassfish.jersey.server.ServerRuntime$Responder.process(ServerRuntime.java:434) ~[?:?]
	at org.glassfish.jersey.server.ServerRuntime$2.run(ServerRuntime.java:329) ~[?:?]
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:271) ~[?:?]
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:267) ~[?:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315) ~[?:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297) ~[?:?]
	at org.glassfish.jersey.internal.Errors.process(Errors.java:267) ~[?:?]
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:317) ~[?:?]
	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:305) ~[?:?]
	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:1154) ~[?:?]
	at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:473) ~[?:?]
	at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:427) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:388) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:341) ~[?:?]
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:228) ~[?:?]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:873) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:542) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceServletHandler.doHandle(HttpServiceServletHandler.java:71) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:146) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:548) ~[87:org.eclipse.jetty.security:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:257) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1700) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:255) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1345) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.HttpServiceContext.doHandle(HttpServiceContext.java:293) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:203) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:480) ~[90:org.eclipse.jetty.servlet:9.4.18.v20190429]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1667) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:201) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1247) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144) ~[89:org.eclipse.jetty.server:9.4.18.v20190429]
	at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:80) ~[193:org.ops4j.pax.web.pax-web-jetty:7.2.10]
	... 15 more
2019-07-30 11:20:09.524 [DEBUG] [ulation.internal.HueEmulationService] - REST request GET xwcVRssJidio22nQS2KflGt0QgOprK6R47iy4QQR/lights/10: 
2019-07-30 11:20:09.535 [DEBUG] [ulation.internal.HueEmulationService] - REST response: {"state":{"bri":254,"ct":500,"on":true,"reachable":true,"mode":"homeautomation","alert":"none"},"type":"Color temperature light","modelid":"LTW001","uniqueid":"44b2a631-db86-44e1-86a6-830d5e885584-10","manufacturername":"Philips","swversion":"66012040","friendsOfHue":true,"colorGamut":"2200K-6500K","hascolor":false,"name":"Schlafzimmer Licht","config":{"archetype":"classicbulb","function":"functional","direction":"omnidirectional"},"capabilities":{"certified":true,"streaming":{"renderer":false,"proxy":false},"control":{}}}
2019-07-30 11:20:15.607 [DEBUG] [ulation.internal.HueEmulationService] - REST request GET e7914080-0d11-462f-b019-7d4eb8bab51d/lights: 
2019-07-30 11:20:15.611 [DEBUG] [ulation.internal.HueEmulationService] - REST response: null

```

--- a/bundles/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/HueEmulationService.java
+++ b/bundles/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/HueEmulationService.java
@@ -97,13 +97,14 @@ public class HueEmulationService implements EventHandler {
                 return;
             }
 
-            InputStream stream = requestContext.getEntityStream();
-            String body = stream != null
-                    ? new BufferedReader(new InputStreamReader(stream)).lines().collect(Collectors.joining("\n"))
-                    : "";
+            // InputStream stream = requestContext.getEntityStream();
+            // String body = stream != null
+            //         ? new BufferedReader(new InputStreamReader(stream)).lines().collect(Collectors.joining("\n"))
+            //         : "";
 
-            logger.debug("REST request {} {}: {}", requestContext.getMethod(), requestContext.getUriInfo().getPath(),
-                    body);
+            // logger.debug("REST request {} {}: {}", requestContext.getMethod(), requestContext.getUriInfo().getPath(),
+            //         body);
+            logger.debug("REST request {} {}", requestContext.getMethod(), requestContext.getUriInfo().getPath());
             logger.debug("REST response: {}", responseContext.getEntity());
         }
 

